lte/gateway/python/magma/pipelined/BUILD.bazel

```diff
@@ -247,7 +248,11 @@ py_library(
 py_library(
     name = "gw_mac_address",
     srcs = ["gw_mac_address.py"],
-    deps = [":ifaces"],
+    deps = [
+        ":ifaces",
+        requirement("getmac"),
+        requirement("scapy"),
```

Comment:
Why did this work previously? Didn't it require scapy before?

Reply:
It's a bit surprising. Maybe it's only imported in a context where scapy is also imported via mandatory_apps, where it was removed now?

Reply:
Right, `mandatory_apps` is the only reverse-dependency of `gw_mac_address`.